src/Batch/Client/Src/GeneratedProtocol/AccountOperations.cs

```diff
@@ -1,58 +1,33 @@
-// Copyright (c) Microsoft and contributors.  All rights reserved.
```

Comment:
Ah... I think I know what happened.  That's a really good catch @cormacpayne I will fix it right now.